tidb-cloud/naming-conventions-for-data-import.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Naming Conventions for Data Import
+summary: Learn about the requirements of file formats for data import.
+---
+
+# Naming Conventions for Data Import
+
+The Data Import feature in TiDB Cloud supports the following file formats: CSV, Parquet, Aurora Snapshot, and SQL File. There are two prerequisites:
+
+- The data files you provide and the schema files for building the database and tables must conform to the naming requirements. If you do not provide the schema file for building the database, TiDB Cloud will create the target database for you according to the default configuration. If you cannot provide data files with the required file names, you can modify the file pattern to perform the import task. Otherwise, TiDB Cloud can not scan the files you need to import.
+- If you can not provide the schema file for building tables, you need to create the target tables in TiDB Cloud in advance.
+
+## Naming conventions for data files and schema files
+
+This section describes the naming conventions for data files and schema files.
+
+### CSV
+
+When you import CSV files, name the schema files and data files as follows:
+
+- DB schema file (optional): `${db_name}-schema-create.sql`
+- Table schema file: `${db_name}.${table_name}-schema.sql`
+- Data file: `${db_name}.${table_name}[.XXXXXX].csv` ([.XXXXXX] is optional)
+
+For example, `import_db.test_table.01.csv`
+
+### Parquet
+
+When you import Parquet files, name the schema files and data files as follows:
+
+- DB schema file(optional): `${db_name}-schema-create.sql`
+- Table schema file: `${db_name}.${table_name}-schema.sql`
+- Data file: `${db_name}.${table_name}[.XXXXXX].{snappy|gz|lzo}.parquet` (`[.XXXXXXX].{snappy|gz|lzo}` is optional )
+
+For example, `import_db.test_table.01.parquet`
+
+### Aurora Snapshot
+
+When you import Aurora Snapshot files, name the schema files and data files as follows:
+
+- DB schema file: `${db_name}-schema-create.sql (optional)`
+- Table schema file: `${db_name}.${table_name}-schema.sql`
+- Data file: All files with the `.parquet` suffix in the `db_name.table_name/` folder.
+
```

Comment:
Any prefix consisting of "a-z, 0-9, - , _ , ." and suffix ".parquet" is fine, such as dbname.tablename.01.parquet.

Reply:
Add this sentence. PTAL.